content/cloud/noops-sql-db.md

```diff
@@ -22,21 +20,10 @@ With NoOps SQL Database using SQLite support in Fermyon Cloud, you can persist r
 
 <iframe width="854" height="480" src="https://www.youtube.com/embed/buwr66oRPbU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
-## Accessing Private Beta
+> Please check that you have [Spin CLI](./cli-reference.md) v1.4 or greater, and [`cloud` plugin](https://github.com/fermyon/cloud-plugin) v0.3.1 or greater installed. 
 
-The database in Fermyon Cloud is currently in private beta. To request access to the private beta, please fill out this short [sign-up form](https://fibsu0jcu2g.typeform.com/to/Brv12FI0#hubspot_utk=xxxxx&hubspot_page_name=xxxxx&hubspot_page_url=xxxxx).
- 
-> Please note that the private beta is limited in space, and all requests cannot be guaranteed. 
+## Service Limitations For the Database in Fermyon Cloud
```

Comment:
The content of this section doesn't really feel like "service limitations" any more?

Reply:
Agreed. I think @tpmccallum has refactored this now.